src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -436,12 +436,7 @@ private function addValidationSection(ArrayNodeDefinition $rootNode)
                     ->info('validation configuration')
                     ->canBeEnabled()
                     ->children()
-                        ->scalarNode('cache')
-                            ->beforeNormalization()
-                                // Can be removed in 3.0, once ApcCache support is dropped
-                                ->ifString()->then(function ($v) { return 'apc' === $v ? 'validator.mapping.cache.apc' : $v; })
```

Comment:
The deprecation seems to have been triggered through the new service in previous versions...
(see https://github.com/symfony/symfony/blob/2.8/src/Symfony/Bundle/FrameworkBundle/Resources/config/validator.xml#L42-L51)

And this service have been removed in 3.0.
I'm getting lost in the operation. I think something went wrong during a merge or this wasn't expected to be as it is right now.

I'm going to make `validator.mapping.cache.apc` an alias of the new `validator.mapping.cache.doctrine.apc` service.